sdks/python/src/opik/api_objects/rest_stream_parser.py

```diff
@@ -0,0 +1,56 @@
+import json
+import logging
+from typing import Iterable, Type, List, Optional, TypeVar
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+T = TypeVar("T")
+
+
+def read_and_parse_stream(stream: Iterable[bytes], item_class: Type[T]) -> List[T]:
```

Comment:
Let's add some unit tests for this function.
We will likely use it later for dataset items and experiment items so let's make sure it's covered.